tests/dml_vdm/dml_vdm.c

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2022, Intel Corporation */
+
+#include <assert.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+#include <libminiasync.h>
+#include <libminiasync-dml.h>
+
+int main(int argc, char *argv[]) {
```

Comment:
please refactor this a bit - I suggest creating one test function `dml-basic-memcpy` that would accept a descriptor and a runtime. Then you can just run that twice with different descriptor implementations. Eliminates needless copy/pasting.

Reply:
Done.